types/yup/index.d.ts

```diff
@@ -89,6 +89,9 @@ export interface NumberSchemaConstructor {
 export interface NumberSchema extends Schema {
     min(limit: number | Ref, message?: string): NumberSchema;
     max(limit: number | Ref, message?: string): NumberSchema;
+    less(limit: number | Ref, message?: string): NumberSchema;
+    more(limit: number | Ref, message?: string): NumberSchema;
```

Comment:
Again, it should be `moreThan `.  see [number.js](https://github.com/jquense/yup/blob/1bc7b56f61d21c8681613acd9b531b9edd420879/src/number.js#L73)